source/minar.cpp

```diff
@@ -251,7 +251,7 @@ int minar::SchedulerData::start(){
 
             if(dispatch_tree.get_num_elements() > 0) {
                 CallbackNode *root = dispatch_tree.get_root();
-                now_plus_tolerance = wrapTime(now + root->tolerance);
+                now_plus_tolerance = wrapTime(now + root->tolerance/2);
```

Comment:
I think that the original purpose of the "double sided tolerance" was to allow an event's tolerance to be applied in each direction relative to the event's scheduled time: [scheduled_time - tolerance, scheduled_time + tolerance]. Since that's not implemented currently, I don't think that multiplying the initial tolerance by 2 gives us any advantage. The proper fix for this would be to actually apply and check the tolerance in both directions.
Cc @autopulated